raiden_libs/network/matrix/room.py

```diff
@@ -13,3 +26,60 @@ def add_ephemeral_listener(self, callback, event_type=None):
 
     def add_state_listener(self, callback, event_type=None):
         super().add_state_listener(geventify_callback(callback), event_type)
+
+    @ttl_cache(ttl=10)
```

Comment:
These decoratores are global state, is it fine to be shared across multiple instances of a raiden node during testing?

Reply:
It isn't a problem. They index their cache by function parameters, including `self`, so multiple instances in a single or multiple processes can coexist. Usually, this cache has some limit to avoid memory overflow, around 128, which is enough for most cases, and on worst will cause some miss on data which were previously cached.